src/components/hr/components.test-pw.tsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-console */
+import React from "react";
+import Hr, { HrProps } from ".";
+import Form from "../form";
+import Textbox from "../textbox";
+import Button from "../button";
+
+export const Default = (props: HrProps) => {
+  return <Hr {...props} />;
+};
+
+export const DifferentSpacing = () => <Hr mt={7} mb={7} />;
+
+export const InsideForm = () => (
+  <Form
+    onSubmit={() => console.log("submit")}
```

Comment:
Comment: we can probably just remove the console logs here rather than disabling the linter